### ISSUE TITLE
Browserhax_cfg.php in the wrong location

### ISSUE THREAD

**Issue by cozyGalvinism** [Cartman123/update_browserhax#1]
```
Warning: include_once(/home/yellows8/browserhax/browserhax_cfg.php): failed to open stream: No such file or directory in /root/3dshacks/browserhax_fright/skater31hax.php on line 5

Warning: include_once(): Failed opening '/home/yellows8/browserhax/browserhax_cfg.php' for inclusion (include_path='.:/usr/share/php:/usr/share/pear') in /root/3dshacks/browserhax_fright/skater31hax.php on line 5

Fatal error: Call to underfined function browserhaxcfg_parsebinparam() in /root/3dshacks/3ds_browserhax_common/3dsbrowserhax_common.php on line 7
```

**Comment by cozyGalvinism** [comment]
Those files need the include_once paths adjusted: http://prntscr.com/d4tgnu

**Comment by cozyGalvinism** [comment]
Added pull request for a quick fix #2
